src/components/views/rooms/RoomSublist2.tsx

```diff
@@ -311,15 +310,42 @@ export default class RoomSublist2 extends React.Component<IProps, IState> {
     }
 
     private renderMenu(): React.ReactElement {
-        // TODO: Get a proper invite context menu, or take invites out of the room list.
-        if (this.props.tagId === DefaultTagID.Invite) {
-            return null;
-        }
-
         let contextMenu = null;
         if (this.state.contextMenuPosition) {
             const isAlphabetical = RoomListStore.instance.getTagSorting(this.props.tagId) === SortAlgorithm.Alphabetic;
             const isUnreadFirst = RoomListStore.instance.getListOrder(this.props.tagId) === ListAlgorithm.Importance;
+
+            // Invites don't get some nonsense options, so only add them if we have to. We do
+            // this with an array instead of a containing div to ensure that the DOM structure
+            // is relatively sane.
+            let otherSections = [];
```

Comment:
this feels like it should just be 

`otherSections = null;`

then `otherSections = <React.Fragment>...</React.Fragment>`
as there is no point dealing with keys